src/explorer/SiteTreeItem.ts

```diff
@@ -98,6 +98,11 @@ export abstract class SiteTreeItem implements IAzureParentTreeItem {
         return await this.siteWrapper.editScmType(client);
     }
 
+    public async deploy(fsPath: string, client: WebSiteManagementClient, outputChannel: OutputChannel, configureSecitonName: string, confirmDeployment: boolean = false): Promise<void> {
```

Comment:
confirmDeployment should default to true

Reply:
suggestion: Some people argue that having booleans defaulting to true is a bad idea (non-intuitive), in which case you could change the flag to dontConfirmDeployment=false or some such.